src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/jpa/repository/ResidentialLocationRepository.kt

```diff
@@ -13,7 +13,6 @@ interface ResidentialLocationRepository : JpaRepository<ResidentialLocation, UUI
   fun findAllByPrisonIdAndParentIsNull(prisonId: String): List<ResidentialLocation>
   fun findAllByPrisonIdAndArchivedIsTrue(prisonId: String): List<ResidentialLocation>
   fun findOneByPrisonIdAndPathHierarchy(prisonId: String, pathHierarchy: String): ResidentialLocation?
-  fun findAllByPrisonIdAndActiveIsTrue(prisonId: String): List<ResidentialLocation>
```

Comment:
are there other parts of the code where this needs removing? or is that already gone but that line had been missed?

Reply:
Its not used and I think I never did use it - was just cleaning up